CODEOWNERS

```diff
@@ -1,4 +1,4 @@
 # These owners will be the default owners for everything in
 # the repo. Unless a later match takes precedence.
 
-*       @rancher-sandbox/team-highlander
+*       @rancher/team-highlander
```

Comment:
don't remember where the team GH handle was listed, in any case we need to update it which should meet following reqs:

`make sure the team @rancher/team-highlander exists, is publicly visible, and has write access to the repository`

Reply:
Used https://github.com/orgs/rancher/teams/highlander instead and works fine, thanks @kkaempf for pointer